desktop/SmesherManager.ts

```diff
@@ -218,6 +242,7 @@ class SmesherManager extends AbstractManager {
         const res = await this.smesherService.stopSmeshing({
           deleteFiles: deleteFiles || false,
         });
+        await this.clearSmesherMetadata();
         const config = await this.loadConfig();
         const genesisId = generateGenesisIDFromConfig(config);
```

Comment:
Since there is appeared `this.genesisID` — we probably don't need to do this long operation of reading, parsing config, and hashing some props ;)